bosh/releases/pre_render_scripts/auctioneer/auctioneer/jobs/patch_auctioneer_ctl.sh

```diff
@@ -1,28 +0,0 @@
-#!/usr/bin/env bash
-
-set -o errexit -o nounset
-
-target="/var/vcap/all-releases/jobs-src/diego/auctioneer/templates/auctioneer_ctl.erb"
```

Comment:
Also same.

Reply:
Squashed history and amended commit message based on these comments.
Commit 5f7cd28c48738106936b93d1ae215ead9c8d412d